src/app/users/views/bulkUserActionsEmails.ejs

```diff
@@ -0,0 +1,89 @@
+<div class="row">
+    <div class="col-8">
+        <h1 class="heading-xlarge">
+            Bulk user actions
+        </h1>
+        <p class="lede">Select all the users you want to affect and the action you wish to perform on each one</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-12">
+        <form method="post" id="form-bulk-user-actions">
+            <input type="hidden" name="_csrf" value="<%= csrfToken %>"/>
+
+            <div class="col-12 %>">
+                <table class="data">
+                    <thead>
+                    <tr>
+                        <th scope="col" class="cwp-15">Name</th>
+                        <th scope="col" class="cwp-35">Email</th>
+                        <th scope="col" class="cwp-25">Organisation</th>
+                        <th scope="col" class="cwp-15">Last Login</th>
+                        <th scope="col" class="cwp-10">Status</th>
+                        <th scope="col" class="cwp-10">Selected</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <% if(locals.users.length === 0) { %>
+                    <tr>
+                        <td colspan="5"><span class="empty-state">No users found</span></td>
+                    </tr>
+                    <% } %>
+                    <% for (let i = 0; i < locals.users.length; i++) { %>
+                    <tr>
+                        <td><%= users[i].name %></td>
+                        <td><span class="breakable"><%= users[i].email %></span></td>
+                        <td>
+                            <% if(users[i].organisation) { %>
+                            <%= users[i].organisation.name %>
+                            <% }else { %>
+                            Unknown
+                            <% } %>
+                        </td>
+                        <td>
+                            <% if(locals.users[i].lastLogin) { %>
+                            <%= locals.moment(locals.users[i].lastLogin).fromNow() %>
+                            <% } else { %>
+                            Never
+                            <% } %>
+                        </td>
+                        <td><%= users[i].status.description %></td>
+                        <td><input type="checkbox" id="user-<%= users[i].id %>" name="user-<%= users[i].id %>" value="<%= users[i].id %>"></td>
+                    </tr>
+                    <% } %>
+                    </tbody>
+                </table>
+            </div>
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
+                  <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                    <h1 class="govuk-fieldset__heading">
+                      Select which actions to apply to all the selected users
+                    </h1>
+                  </legend>
+                  <div class="govuk-checkboxes" data-module="govuk-checkboxes">
```

Comment:
Both of these checkboxes here should look like the other checkboxes in the application, I believe the current classes and set-up here are for the new GDS version, whereas in support they need to be styled like the following: https://github.com/DFE-Digital/login.dfe.support/blob/f806cfc256b737d95da352d363a40c05877bbd0e/src/app/users/views/associateServices.ejs#L40-L49

Alternatively, if you want to use the new GDS design, you can hard-code the layout when rendering these pages to use the new GDS compliant layout: https://github.com/DFE-Digital/login.dfe.support/blob/f806cfc256b737d95da352d363a40c05877bbd0e/src/app/sharedViews/layoutNew.ejs but that may cause additional issues 😄